src/ingestor.ts

```diff
@@ -96,13 +96,13 @@ export default async function ingestor(req) {
 
   if (type === 'settings') payload = JSON.parse(message.settings);
 
-  if (type === 'proposal')
+  if (type === 'proposal') {
     payload = {
       name: message.title,
       body: message.body,
       discussion: message.discussion || '',
       choices: message.choices,
-      start: message.start,
+      start: message.start > ts ? message.start : ts,
```

Comment:
@Todmy I think It should throw an error instead of modifying the `start` users may complain comparing with IPFS file